### DISCUSSION TITLE
Filter differences between a personal profile and an organization one

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#108266] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My profile:
<img width="313" alt="Screenshot 2024-02-14 alle 08 22 55" src="https://github.com/community/community/assets/20476002/3d7e7d20-ad6c-40b9-bb48-7759024a9b6e">

<img width="326" alt="Screenshot 2024-02-14 alle 08 23 31" src="https://github.com/community/community/assets/20476002/7d3df8b1-76cb-408a-b45c-4d98e291256e">

Profile Organization:
<img width="207" alt="Screenshot 2024-02-14 alle 08 25 39" src="https://github.com/community/community/assets/20476002/7bd075b1-9110-4d93-aeef-04b2c3b36640">

As you can see from the images there is a lot of difference in filters that can be applied to one and the other.

The personal profile is missing:
- ascending or descending sorting for fields

The organization profile is missing:
- sorting by stars
- search for all those who are of a specific language

On the personal profile it is called `Last updated`, on the organization profile it is called `Last pushed`.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Angelk90 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
